server/index.html

```diff
@@ -3,38 +3,146 @@
-      <div class="game-grid-item" id="grid-1" style="border: 1px solid black; height: 50px; width: 50px;">1</div>
-      <div class="game-grid-item" id="grid-2" style="border: 1px solid black; height: 50px; width: 50px;">2</div>
-      <div class="game-grid-item" id="grid-3" style="border: 1px solid black; height: 50px; width: 50px;">3</div>
-      <div class="game-grid-item" id="grid-4" style="border: 1px solid black; height: 50px; width: 50px;">4</div>
-      <div class="game-grid-item" id="grid-5" style="border: 1px solid black; height: 50px; width: 50px;">5</div>
-      <div class="game-grid-item" id="grid-6" style="border: 1px solid black; height: 50px; width: 50px;">6</div>
-      <div class="game-grid-item" id="grid-7" style="border: 1px solid black; height: 50px; width: 50px;">7</div>
-      <div class="game-grid-item" id="grid-8" style="border: 1px solid black; height: 50px; width: 50px;">8</div>
-      <div class="game-grid-item" id="grid-9" style="border: 1px solid black; height: 50px; width: 50px;">9</div>
+  </header>
+  <div class="container">
+    <div class="game">
+      <div class="prompts">
+        <form class="prompts-form">
+          <div class="prompts-wrapper">
+            <section class="prompts-players">
+              <h2 class="prompts-player-header">Players</h2>
+              <div class="prompts-wrapper-inner">
+                <label class="prompts-radio-label" for="player-num-1">1</label>
+                <input class="prompts-radio" type="radio" id="1-player" name="player-num" value="1-player" checked>
+
+                <label class="prompts-radio-label" for="player-num-2">2</label>
+                <input class="prompts-radio" type="radio" id="2-player" name="player-num" value="2-player">
+              </div>
+            </section>
+            <section class="prompts-team">
+              <h2 class="prompts-team-header">Team</h2>
+              <div class="prompts-wrapper-inner">
+                <label class="prompts-radio-label" for="team-o">O</label>
+                <input class="prompts-radio" type="radio" id="team-o" name="player-team" value="team-o" checked>
+
+                <label class="prompts-radio-label" for="team-x">X</label>
+                <input class="prompts-radio" type="radio" id="team-x" name="player-team" value="team-x">
+              </div>
+            </section>
+          </div>
+          <div class="prompts-submit">
+            <button
+            type="button" class="prompts-form-submit" onClick="startGame()">Start Game!</button>
+          </div>
+        </form>
+      </div>
+
+      <div class="game-header" style="display: flex; justify-content: center;">
+        <h2 class="game-header-player" id="current-player" style="margin-right: 50px;">Current Player</h2>
+        <h2 class="game-header-team" id="current-team">Team</h2>
+      </div>
+      <div class="game-grid-wrapper">
```

Comment:
When I played it, it wasn't obvious that the grid was not active yet. Might be best to hide it entirely until you've picked your game type.

Reply:
(and probably entirely-hide the radio buttons when you're mid-game?)